gradle.properties

```diff
@@ -2,7 +2,7 @@
 # This is required to provide enough memory for the Minecraft decompilation process.
 org.gradle.jvmargs=-Xmx3G
 org.gradle.daemon=false
-MC_VERSION=1.16.1
-FORGE_VERSION=32.0.75
+MC_VERSION=1.16.3
+FORGE_VERSION=34.1.5
 MCP_CHANNEL=snapshot
-MCP_MAPPINGS=20200514-1.16
\ No newline at end of file
+MCP_MAPPINGS=20200916-1.16.2
```

Comment:
This probably needs to be updated too, but I don't know how =S. Where do you see the mappings from version 1.17.1?